ltr/concourse/pipeline.yaml

```diff
@@ -0,0 +1,266 @@
+# common configuration
+task-config: &task-config
+  platform: linux
+  image_resource:
+    type: docker-image
+    source:
+      repository: python
+      tag: 3.7
+
+# proper configuration begins
+resource_types:
+- name: cron-resource
+  type: docker-image
+  source:
+    repository: cftoolsmiths/cron-resource
+    tag: latest
+
+resources:
+- name: at-10pm
+  type: cron-resource
+  source:
+    expression: "00 22 * * *"
+    location: "Europe/London"
+- name: search-api-git
+  type: git
+  source:
+    uri: https://github.com/alphagov/search-api.git
+- name: integration-training-data
+  type: s3
+  source:
+    bucket: ((readonly_private_bucket_name))
+    region_name: eu-west-2
+    regexp: search-learn-to-rank/integration-training-data-(.*).txt
+- name: staging-training-data
+  type: s3
+  source:
+    bucket: ((readonly_private_bucket_name))
+    region_name: eu-west-2
+    regexp: search-learn-to-rank/staging-training-data-(.*).txt
+- name: production-training-data
+  type: s3
+  source:
+    bucket: ((readonly_private_bucket_name))
+    region_name: eu-west-2
+    regexp: search-learn-to-rank/production-training-data-(.*).txt
+- name: integration-model
+  type: s3
+  source:
+    bucket: ((readonly_private_bucket_name))
+    region_name: eu-west-2
+    regexp: search-learn-to-rank/integration-model-(.*).txt
+- name: staging-model
+  type: s3
+  source:
+    bucket: ((readonly_private_bucket_name))
+    region_name: eu-west-2
+    regexp: search-learn-to-rank/staging-model-(.*).txt
+- name: production-model
+  type: s3
+  source:
+    bucket: ((readonly_private_bucket_name))
+    region_name: eu-west-2
+    regexp: search-learn-to-rank/production-model-(.*).txt
+
+jobs:
+- name: integration-fetch
```

Comment:
We might want to make the fetch jobs (and other jobs) resilient to the Concourse worker being terminated.  Or maybe just clicking the button to run it again on failure is good enough.